backend/dataall/modules/redshift_datasets/services/redshift_dataset_service.py

```diff
@@ -45,6 +46,46 @@
 
 
 class RedshiftDatasetService:
```

Comment:
nit: it might be nice to have an interface for DatasetServices as well

Reply:
Agree. In this PR I did not want to mix things too much (I am already creating validation code + moving code around). There is an item in the next steps to remove duplications, I'll reference this comment there https://github.com/data-dot-all/dataall/issues/1509